src/lib/fetch-document-list.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-console */
+const { info } = require('../lib/log');
+const fs = require('../lib/sync-fs');
+const INVALID_DOC_TYPES = ['district_hospital', 'health_center', 'clinic', 'person', 'user', 'user-settings', 'contact'];
```

Comment:
Aren't these "VALID_DOC_TYPES" ? I looks like we're throwing errors for all docs that don't have a type in this list, so these sound to me like they are valid doc types, instead.